### ISSUE TITLE
Runtime Exception: Broken stub format

### ISSUE THREAD

**Issue by zamhassam** [giraud/reasonml-idea-plugin#355]
plugin version: 
0.102.2 - 2021.2
### Description
The IDE keeps raising this when I try to do a Go To Definition (Ocaml code) on anything.  Any help you can give would be most appreciated.

```
java.lang.RuntimeException: Broken stub format, most likely version of OCAML_FILE (ocaml.FILE) was not updated after serialization changes

	at com.intellij.psi.stubs.LazyStubList.instantiateStub(LazyStubList.java:96)
	at com.intellij.psi.stubs.LazyStubList.get(LazyStubList.java:58)
	at com.intellij.psi.stubs.LazyStubList.get(LazyStubList.java:18)
	at com.intellij.psi.stubs.StubSpine.getStubPsi(StubSpine.java:45)
	at com.intellij.psi.stubs.StubProcessingHelperBase.getStubPsi(StubProcessingHelperBase.java:98)
	at com.intellij.psi.stubs.StubProcessingHelperBase.processStubsInFile(StubProcessingHelperBase.java:71)
	at com.intellij.psi.stubs.StubIndexImpl.lambda$processElements$2(StubIndexImpl.java:287)
	at com.intellij.psi.stubs.StubIndexImpl.processElements(StubIndexImpl.java:333)
	at com.intellij.psi.stubs.StubIndex.processElements(StubIndex.java:51)
	at com.reason.lang.core.psi.reference.ORElementResolver.lambda$new$4(ORElementResolver.java:35)
	at com.intellij.util.io.keyStorage.AppendableStorageBackedByResizableMappedFile.lambda$processAll$3(AppendableStorageBackedByResizableMappedFile.java:109)
	at com.intellij.util.io.PagedFileStorage.lambda$readInputStream$1(PagedFileStorage.java:114)
	at com.intellij.util.io.PagedFileStorage.useChannel(PagedFileStorage.java:130)
	at com.intellij.util.io.PagedFileStorage.readInputStream(PagedFileStorage.java:112)
	at com.intellij.util.io.ResizeableMappedFile.readInputStream(ResizeableMappedFile.java:261)
	at com.intellij.util.io.keyStorage.AppendableStorageBackedByResizableMappedFile.processAll(AppendableStorageBackedByResizableMappedFile.java:96)
	at com.intellij.util.io.PersistentEnumeratorBase.iterateData(PersistentEnumeratorBase.java:426)
	at com.intellij.util.io.PersistentMapImpl.processKeys(PersistentMapImpl.java:464)
	at com.intellij.util.indexing.impl.ValueContainerMap.processKeys(ValueContainerMap.java:69)
	at com.intellij.util.indexing.impl.MapIndexStorage.doProcessKeys(MapIndexStorage.java:365)
	at com.intellij.util.indexing.impl.storage.VfsAwareMapIndexStorage.processKeys(VfsAwareMapIndexStorage.java:122)
	at com.intellij.util.indexing.impl.storage.TransientChangesIndexStorage.processKeys(TransientChangesIndexStorage.java:139)
	at com.intellij.util.indexing.impl.storage.VfsAwareMapReduceIndex.lambda$processAllKeys$3(VfsAwareMapReduceIndex.java:274)
	at com.intellij.util.ConcurrencyUtil.withLock(ConcurrencyUtil.java:234)
	at com.intellij.util.indexing.impl.storage.VfsAwareMapReduceIndex.processAllKeys(VfsAwareMapReduceIndex.java:273)
	at com.intellij.indexing.composite.CompositeInvertedIndexBase.processAllKeys(CompositeInvertedIndexBase.java:106)
	at com.intellij.psi.stubs.StubIndexImpl.lambda$processAllKeys$6(StubIndexImpl.java:440)
	at com.intellij.util.indexing.IndexUpToDateCheckIn.disableUpToDateCheckIn(IndexUpToDateCheckIn.kt:13)
	at com.intellij.util.indexing.FileBasedIndexImpl.disableUpToDateCheckIn(FileBasedIndexImpl.java:743)
	at com.intellij.psi.stubs.StubIndexImpl.lambda$processAllKeys$7(StubIndexImpl.java:439)
	at com.intellij.util.indexing.diagnostic.IndexAccessValidator.validate(IndexAccessValidator.java:42)
	at com.intellij.psi.stubs.StubIndexImpl.processAllKeys(StubIndexImpl.java:439)
	at com.intellij.psi.stubs.StubIndex.processAllKeys(StubIndex.java:68)
	at com.reason.lang.core.psi.reference.ORElementResolver.lambda$new$5(ORElementResolver.java:34)
	at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:39)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$3(CachedValueBase.java:227)
	at com.intellij.util.CachedValueBase.computeData(CachedValueBase.java:42)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$4(CachedValueBase.java:227)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:43)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:68)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:228)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:28)
	at com.reason.lang.core.psi.reference.ORElementResolver$ResolutionsImpl.addIncludesEquivalence(ORElementResolver.java:210)
	at com.reason.lang.core.psi.reference.PsiLowerSymbolReference.multiResolve(PsiLowerSymbolReference.java:94)
	at com.reason.lang.core.psi.reference.ORMultiSymbolReference.resolve(ORMultiSymbolReference.java:21)
	at com.intellij.codeInsight.TargetElementUtilBase.getReferencedElement(TargetElementUtilBase.java:177)
	at com.intellij.codeInsight.TargetElementUtilBase.doGetReferenceOrReferencedElement(TargetElementUtilBase.java:165)
	at com.intellij.codeInsight.TargetElementUtilBase.getReferenceOrReferencedElement(TargetElementUtilBase.java:204)
	at com.intellij.codeInsight.TargetElementUtilBase.getReferencedElement(TargetElementUtilBase.java:248)
	at com.intellij.codeInsight.TargetElementUtilBase.doFindTargetElement(TargetElementUtilBase.java:227)
	at com.intellij.codeInsight.TargetElementUtilBase.findTargetElement(TargetElementUtilBase.java:292)
	at com.intellij.codeInsight.TargetElementUtil.findTargetElement(TargetElementUtil.java:147)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorPsiDataProvider.getPsiElementIn(TextEditorPsiDataProvider.java:226)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorPsiDataProvider.getSlowData(TextEditorPsiDataProvider.java:139)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorPsiDataProvider.lambda$getData$0(TextEditorPsiDataProvider.java:93)
	at com.intellij.ide.impl.DataManagerImpl.getSlowData(DataManagerImpl.java:137)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataRule$2(DataManagerImpl.java:107)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataRule$4(DataManagerImpl.java:112)
	at com.intellij.ide.impl.DataManagerImpl.getDataFromProvider(DataManagerImpl.java:73)
	at com.intellij.ide.impl.DataManagerImpl.getDataFromProvider(DataManagerImpl.java:52)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataFromProvider$1(DataManagerImpl.java:73)
	at com.intellij.openapi.actionSystem.DataKey.getData(DataKey.java:68)
	at com.intellij.ide.impl.dataRules.NavigatableRule.getData(NavigatableRule.java:39)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataRule$4(DataManagerImpl.java:116)
	at com.intellij.ide.impl.DataManagerImpl.getDataFromProvider(DataManagerImpl.java:73)
	at com.intellij.ide.impl.DataManagerImpl.getDataFromProvider(DataManagerImpl.java:52)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataFromProvider$1(DataManagerImpl.java:73)
	at com.intellij.openapi.actionSystem.DataKey.getData(DataKey.java:68)
	at com.intellij.ide.impl.dataRules.NavigatableArrayRule.getData(NavigatableArrayRule.java:12)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataRule$4(DataManagerImpl.java:116)
	at com.intellij.ide.impl.DataManagerImpl.getDataFromProvider(DataManagerImpl.java:73)
	at com.intellij.openapi.actionSystem.impl.PreCachedDataContext.getData(PreCachedDataContext.java:126)
	at com.intellij.openapi.actionSystem.DataContext.getData(DataContext.java:42)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.ensureSlowDataKeysPreCached(ActionUpdater.java:355)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$14(ActionUpdater.java:279)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$16(ActionUpdater.java:305)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1078)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$17(ActionUpdater.java:305)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:161)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$18(ActionUpdater.java:301)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:183)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.runUnderDisposeAwareIndicator(BackgroundTaskUtil.java:270)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$19(ActionUpdater.java:300)
	at com.intellij.codeWithMe.ClientId$Companion.withClientId(ClientId.kt:135)
	at com.intellij.codeWithMe.ClientId.withClientId(ClientId.kt)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$20(ActionUpdater.java:298)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:216)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:27)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:195)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:213)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:184)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.RuntimeException: Broken stub format, most likely version of OCAML_FILE (ocaml.FILE) was not updated after serialization changes

	at com.intellij.psi.stubs.LazyStubList.instantiateStub(LazyStubList.java:96)
	at com.intellij.psi.stubs.LazyStubList.get(LazyStubList.java:58)
	at com.intellij.psi.stubs.LazyStubList.instantiateStub(LazyStubList.java:87)
	... 102 more
Caused by: java.lang.AssertionError: Stub serializer/deserialize mismatch for StubElementType: name = C_MODULE_DECLARATION, language = Language: OCaml
	at com.intellij.psi.stubs.LazyStubData.deserializeStub(LazyStubList.java:135)
	at com.intellij.psi.stubs.LazyStubList.instantiateStub(LazyStubList.java:88)
	... 104 more
```

**Comment by giraud** [comment]
try to invalidate the caches and restart.

**Comment by zamhassam** [comment]
No luck unfortunately.

**Comment by zamhassam** [comment]
[ch4.zip](https://github.com/giraud/reasonml-idea-plugin/files/7249871/ch4.zip)
Of course!  Please ignore the code quality and there may be something stupid I have done with the set up (I am currently learning Ocaml).

**Comment by giraud** [comment]
I don't see a problem.
I have published a beta version in which I have updated all serialization versions of file stubs. Can you try it ?
The repository channel is Beta. You need to add it as a custom repository like explained here : https://www.jetbrains.com/help/idea/managing-plugins.html#repos, and using this url : https://plugins.jetbrains.com/plugins/beta/list

**Comment by TimOrtel** [comment]
plugin version: 0.106-2022.1

### Description

The plugin displays an error message on about every action I try to perform. I am running a rather big dune project. 

`java.lang.RuntimeException: Broken stub format, most likely version of OCAML_FILE (ocaml.FILE) was not updated after serialization changes

	at com.intellij.psi.stubs.LazyStubList.instantiateStub(LazyStubList.java:96)
	at com.intellij.psi.stubs.LazyStubList.get(LazyStubList.java:58)
	at com.intellij.psi.stubs.LazyStubList.get(LazyStubList.java:18)
	at com.intellij.psi.stubs.StubSpine.getStubPsi(StubSpine.java:45)
	at com.intellij.psi.stubs.StubProcessingHelperBase.getStubPsi(StubProcessingHelperBase.java:98)
	at com.intellij.psi.stubs.StubProcessingHelperBase.processStubsInFile(StubProcessingHelperBase.java:71)
	at com.intellij.psi.stubs.StubIndexEx.lambda$processElements$4(StubIndexEx.java:140)
	at com.intellij.psi.stubs.StubIndexEx.processElements(StubIndexEx.java:187)
	at com.intellij.psi.stubs.StubIndex.processElements(StubIndex.java:50)
	at com.reason.lang.core.psi.reference.ORElementResolver.lambda$new$4(ORElementResolver.java:36)
	at com.intellij.util.io.PersistentEnumeratorBase.lambda$iterateData$1(PersistentEnumeratorBase.java:410)
	at com.intellij.util.io.keyStorage.AppendableStorageBackedByResizableMappedFile.lambda$processAll$3(AppendableStorageBackedByResizableMappedFile.java:115)
	at com.intellij.util.io.PagedFileStorage.lambda$readInputStream$1(PagedFileStorage.java:115)
	at com.intellij.util.io.PagedFileStorage.useChannel(PagedFileStorage.java:144)
	at com.intellij.util.io.PagedFileStorage.readInputStream(PagedFileStorage.java:113)
	at com.intellij.util.io.ResizeableMappedFile.readInputStream(ResizeableMappedFile.java:268)
	at com.intellij.util.io.keyStorage.AppendableStorageBackedByResizableMappedFile.processAll(AppendableStorageBackedByResizableMappedFile.java:102)
	at com.intellij.util.io.PersistentEnumeratorBase.doIterateData(PersistentEnumeratorBase.java:422)
	at com.intellij.util.io.PersistentEnumeratorBase.iterateData(PersistentEnumeratorBase.java:410)
	at com.intellij.util.io.PersistentMapImpl.processKeys(PersistentMapImpl.java:488)
	at com.intellij.util.indexing.impl.ValueContainerMap.processKeys(ValueContainerMap.java:73)
	at com.intellij.util.indexing.impl.MapIndexStorage.doProcessKeys(MapIndexStorage.java:368)
	at com.intellij.util.indexing.impl.storage.VfsAwareMapIndexStorage.processKeys(VfsAwareMapIndexStorage.java:125)
	at com.intellij.util.indexing.impl.storage.TransientChangesIndexStorage.processKeys(TransientChangesIndexStorage.java:139)
	at com.intellij.util.indexing.storage.MapReduceIndexBase.lambda$processAllKeys$0(MapReduceIndexBase.java:52)
	at com.intellij.util.ConcurrencyUtil.withLock(ConcurrencyUtil.java:233)
	at com.intellij.util.indexing.storage.MapReduceIndexBase.processAllKeys(MapReduceIndexBase.java:51)
	at com.intellij.indexing.composite.CompositeInvertedIndexBase.processAllKeys(CompositeInvertedIndexBase.java:120)
	at com.intellij.psi.stubs.StubIndexEx.lambda$processAllKeys$8(StubIndexEx.java:284)
	at com.intellij.util.indexing.IndexUpToDateCheckIn.disableUpToDateCheckIn(IndexUpToDateCheckIn.kt:15)
	at com.intellij.util.indexing.FileBasedIndexEx.disableUpToDateCheckIn(FileBasedIndexEx.java:785)
	at com.intellij.psi.stubs.StubIndexEx.lambda$processAllKeys$9(StubIndexEx.java:283)
	at com.intellij.util.indexing.diagnostic.IndexAccessValidator.validate(IndexAccessValidator.java:42)
	at com.intellij.psi.stubs.StubIndexEx.processAllKeys(StubIndexEx.java:283)
	at com.intellij.psi.stubs.StubIndex.processAllKeys(StubIndex.java:67)
	at com.reason.lang.core.psi.reference.ORElementResolver.lambda$new$5(ORElementResolver.java:35)
	at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:39)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$3(CachedValueBase.java:227)
	at com.intellij.util.CachedValueBase.computeData(CachedValueBase.java:42)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$4(CachedValueBase.java:227)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:114)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:44)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:68)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:228)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:28)
	at com.reason.lang.core.psi.reference.ORElementResolver$ResolutionsImpl.addIncludesEquivalence(ORElementResolver.java:211)
	at com.reason.lang.core.psi.reference.PsiLowerSymbolReference.multiResolve(PsiLowerSymbolReference.java:94)
	at com.reason.lang.core.psi.reference.ORMultiSymbolReference.resolve(ORMultiSymbolReference.java:21)
	at com.intellij.codeInsight.TargetElementUtilBase.getReferencedElement(TargetElementUtilBase.java:177)
	at com.intellij.codeInsight.TargetElementUtilBase.doGetReferenceOrReferencedElement(TargetElementUtilBase.java:165)
	at com.intellij.codeInsight.TargetElementUtilBase.getReferenceOrReferencedElement(TargetElementUtilBase.java:210)
	at com.intellij.codeInsight.TargetElementUtilBase.getReferencedElement(TargetElementUtilBase.java:254)
	at com.intellij.codeInsight.TargetElementUtilBase.doFindTargetElement(TargetElementUtilBase.java:233)
	at com.intellij.codeInsight.TargetElementUtilBase.findTargetElement(TargetElementUtilBase.java:298)
	at com.intellij.codeInsight.TargetElementUtil.findTargetElement(TargetElementUtil.java:147)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorPsiDataProvider.getPsiElementIn(TextEditorPsiDataProvider.java:226)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorPsiDataProvider.getSlowData(TextEditorPsiDataProvider.java:139)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorPsiDataProvider.lambda$getData$0(TextEditorPsiDataProvider.java:93)
	at com.intellij.ide.impl.DataManagerImpl.getSlowData(DataManagerImpl.java:153)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataRule$2(DataManagerImpl.java:121)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataRule$4(DataManagerImpl.java:126)
	at com.intellij.ide.impl.DataManagerImpl.getDataFromProvider(DataManagerImpl.java:77)
	at com.intellij.ide.impl.DataManagerImpl.getDataFromProvider(DataManagerImpl.java:56)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataFromProvider$1(DataManagerImpl.java:77)
	at com.intellij.openapi.actionSystem.DataKey.getData(DataKey.java:68)
	at com.intellij.ide.impl.dataRules.NavigatableRule.getData(NavigatableRule.java:39)
	at com.intellij.ide.impl.DataManagerImpl.getRulesData(DataManagerImpl.java:139)
	at com.intellij.ide.impl.DataManagerImpl.lambda$getDataRule$4(DataManagerImpl.java:127)
	at com.intellij.ide.impl.DataManagerImpl.getDataFromProvider(DataManagerImpl.java:77)
	at com.intellij.openapi.actionSystem.impl.PreCachedDataContext.getData(PreCachedDataContext.java:182)
	at com.intellij.openapi.actionSystem.DataContext.getData(DataContext.java:42)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.ensureSlowDataKeysPreCached(ActionUpdater.java:403)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$callAction$5(ActionUpdater.java:185)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1152)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.callAction(ActionUpdater.java:185)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.updateActionReal(ActionUpdater.java:126)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$new$0(ActionUpdater.java:111)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.update(ActionUpdater.java:628)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandGroupChild(ActionUpdater.java:472)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$doExpandActionGroup$20(ActionUpdater.java:451)
	at com.intellij.util.containers.ContainerUtil.concat(ContainerUtil.java:1400)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doExpandActionGroup(ActionUpdater.java:451)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandGroupChild(ActionUpdater.java:529)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$doExpandActionGroup$20(ActionUpdater.java:451)
	at com.intellij.util.containers.ContainerUtil.concat(ContainerUtil.java:1400)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doExpandActionGroup(ActionUpdater.java:451)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandActionGroup(ActionUpdater.java:259)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$12(ActionUpdater.java:320)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$14(ActionUpdater.java:343)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1152)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$15(ActionUpdater.java:343)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:158)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$16(ActionUpdater.java:339)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:189)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:176)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.runUnderDisposeAwareIndicator(BackgroundTaskUtil.java:365)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$17(ActionUpdater.java:338)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:241)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:31)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:214)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:203)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.AssertionError: Stub serializer/deserialize mismatch for StubElementType: name = C_INCLUDE, language = Language: OCaml
	at com.intellij.psi.stubs.LazyStubData.deserializeStub(LazyStubList.java:135)
	at com.intellij.psi.stubs.LazyStubList.instantiateStub(LazyStubList.java:88)
	... 113 more`

**Comment by giraud** [comment]
THank you for reporting. this kind of error is difficult to debug unfortunately.

**Comment by TimOrtel** [comment]
Is there something I can change in my setup? Maybe I have a wrong Ocaml version?

**Comment by giraud** [comment]
No, the problem must be how stubs are built. but I don't know why yet.

**Comment by giraud** [comment]
I am working to update parsers deeply right now and can't work on that bug yet. I need to have a stable branch first.

**Comment by giraud** [comment]
can you try with 0.107 ?

**Comment by giraud** [comment]
should be fixed in 0.109
